lib/galaxy/tools/__init__.py

```diff
@@ -1687,6 +1687,7 @@ def check_and_update_param_values_helper( self, inputs, values, trans, messages,
                             messages[ input.name ].append( rep_dict )
                             self.check_and_update_param_values_helper( input.inputs, {}, trans, rep_dict, context, rep_prefix, allow_workflow_parameters=allow_workflow_parameters )
                 elif isinstance( input, Section ):
+                    messages[ input.name ] = {}
```

Comment:
This line and 1720 should probably be double-checked by @blankenberg and @guerler which modified them a few times between the 3rd and 14th of December.